ledger/ledger-api-common/src/test/suite/scala/com/digitalasset/ledger/api/validation/TransactionServiceRequestValidatorTest.scala

```diff
@@ -231,7 +231,7 @@ class TransactionServiceRequestValidatorTest
           filtersByParty should have size 1
           inside(filtersByParty.headOption.value) { case (p, filters) =>
             p shouldEqual party
-            filters shouldEqual domain.Filters(Some(domain.InclusiveFilters(Set())))
```

Comment:
do we plan to add more tests, here? when?

Reply:
Currently this is just an update of the filter domain representation and validation has not been changed. This test will be changed together with an API change.